sympy/diffgeom/diffgeom.py

```diff
@@ -321,85 +514,133 @@ def coord_tuple_transform_to(self, to_sys, coords):
 
     def jacobian(self, to_sys, coords):
         """Return the jacobian matrix of a transformation."""
+        SymPyDeprecationWarning(
```

Comment:
Although I acknowledge that the term `jacobian` was not used consistently across sympy as well as the general scientific community, I'm not sure this had been necessary to deprecate

Reply:
I'm convinced. Instead, I made `jacobian_matrix` as an alias of `jacobian`.